_posts/2017-11-08-cost-of-drupal-7.md

```diff
@@ -0,0 +1,194 @@
+
+If you're made of time today, check out our [other articles we've written about performance](/blog/tag/performance) for a deeper dive into this broad and complex topic.
+
+### WIIFM? People.
+
+Anyone knows that behind any powerful movement are powerful people. To quote Dries, as I did in my Drupalcon talk in New Orleans:
+
+> fostering the Drupal community is actually more important than just managing the code base.
+
+Also, atop the Drupal.org homepage [used to read](https://events.drupal.org/neworleans2016/sessions/stay-community)
+
+> Come for the code, stay for the community.
+
+Without needing to resonate with all the warm and fuzzies that many within the community do, these sentiments show the richness and value of the Drupal community. And that rich community, not out of neglect, but rather necessity, has moved on from Drupal 7. Top designers, developers, and strategists are working on few Drupal 7 projects these days, and most would prefer to move on. For those who work with web designers and engineers (or [used to be one like me](https://www.drupal.org/u/chrisarusso)), you know that they often have an insatiable appetite for learning, and want to do that with increasingly relevant tools to their growth and output. Sticking to dated software is an effective way to weed out the best and brightest.
+
+#### The Improved Developer Experience (DX) of Drupal 8
+
+Just like happy customers tend to be repeat customers, [happier developers](http://fortune.com/2015/10/29/happy-productivity-work/) [also produce returns](https://www.fastcompany.com/3048751/happy-employees-are-12-more-productive-at-work); they're [more productive](http://www.snacknation.com/blog/employee-happiness/).
+
+There are a few improvements in Drupal 8 that make life _substantially_ better for developers. The [Configuration Management Initiative](https://www.drupal.org/docs/8/configuration-management/managing-your-sites-configuration) was a boon to developers who struggled with a module called "features" which was not designed to do what most of us used it for. The CMI addresses the previous workaround, rife with inconsistencies, of moving site configuration from development to staging and production environments. Although it may seem trivial, developers _love this better system_ in Drupal 8, and it means more efficient development, therefore higher ROI.
+
+<!-- @todo: Get team feedback on this topic. Include composer too? It seems with both there have been some difficulties. Anything else? -->
```

Comment:
Perhaps just reference that with Composer it is now easier than ever for developers to include external libraries in their projects. You could also link to our Composer blog posts: https://savaslabs.com/blog/tag/composer/

Reply:
And to answer your question Chris, I appreciate the new configuration management system. And I prefer Drupal 8 development because it adheres to modern best practices soooo much better. Best practices are best practices for a reason. Adopting these best practices (a) generally, make development much easier (b) makes Drupal more flexible, and (c) make me feel like I'm learning more transferable skills rather than just "Drupalisms"